### PR TITLE
Added Django-Integrations Package

### PR DESCRIPTION
**Pull request by rhimmelbauer**
Notes:
- Added django-integrations
- Added vouchery integration implementation. 

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: develop/develop/settings.py [hunk 1]
```diff
@@ -160,8 +161,8 @@
 
 # Vendor Promo Settings
 VENDOR_PROMO_PROCESSOR = os.getenv("VENDOR_PROMO_PROCESSOR", "base.PromoProcessorBase")
-VENDOR_PROMO_PROCESSOR_URL = os.getenv("VENDOR_PROMO_PROCESSOR_URL")
-VENDOR_PROMO_PROCESSOR_BARRER_KEY = os.getenv("VENDOR_PROMO_PROCESSOR_BARRER_KEY")
+VENDOR_PROMO_PROCESSOR_URL = os.getenv("VENDOR_PROMO_PROCESSOR_URL", None)
+VENDOR_PROMO_PROCESSOR_BARRER_KEY = os.getenv("VENDOR_PROMO_PROCESSOR_BARRER_KEY", None)
```
Comment (mgcamisa) [inline develop/develop/settings.py:165]: definitely not a blocker, but os.getenv already returns None as its default

File: vendorpromo/forms.py [hunk 2]
```diff
@@ -63,3 +64,17 @@ class Meta:
         )
     }
 )
+
+class VoucheryIntegrationForm(forms.ModelForm):
+    class Meta:
+        model = Credential
+        fields = ['client_url', 'private_key']
+        labels = {
+            'client_url': "Vouchery URL",
```
Comment (mgcamisa) [inline vendorpromo/forms.py:73]: do you need to translate these?

**Review by mgcamisa — APPROVED** [review]

File: develop/develop/settings.py (hunk 1 above)
Comment (jared-hardy) [reply develop/develop/settings.py:165]: @mgcamisa this is known, but I prefer this explicit style of default definition, as its more visually useful in `if env_var_defined:` style use cases, and more resilient against upstream library defaults changes over time.

Comment (rhimmelbauer) [reply develop/develop/settings.py:165]: Yes, that is exactly why I did it.

File: vendorpromo/forms.py (hunk 2 above)
Comment (rhimmelbauer) [reply vendorpromo/forms.py:73]: I would not add it

**Pull request merged**